prometheus-ksonnet/lib/prometheus-config.libsonnet

```diff
@@ -5,8 +5,8 @@
     },
 
     rule_files: [
-      'alerts.rules',
-      'recording.rules',
+      'alerts/rules',
+      'recording/rules',
```

Comment:
Shouldn't this be `alerts/alerts.rules` and `recording/recording.rules`?

Reply:
right again! Had to think about this one though.